### ISSUE TITLE
Draft initial Twine story

### ISSUE THREAD

**Issue by CliffordAnderson** [HeardLibrary/twine-for-wikicite#5]

**Comment by marjans74** [comment]
Working with the initial story, which was a possible outline, I developed the Journal Article passage to show how it might progress for the user.  The following steps were added to the passage: Search, Create, Instance Of, Add Statement, Publish, and Author.  On each step text was added to elaborate that step, emphasizing the Property and Value that comprised a statement.  I continued with additional steps such as publication date, journal title, author, etc.  I encountered an unsolvable issue with Twine.  I got an error message, stating save failed, and item was destroyed.  After informing Andy, on his suggestion I sent the archive version to Steve, and Sarah to review as the cause of the issue, and how to resolve it exceeds the scope of my working knowledge of Twine.  Steve did say that the game progressed as expected to the point where it abruptly ends but had nothing further to add at this time.  I was unable to load the Twine file to GitHub, receiving an error that the file format was incompatible with GitHub.
